Tests/RabbitMq/BaseAmqpTest.php

```
@@ -9,7 +9,7 @@
 class BaseAmqpTest extends \PHPUnit_Framework_TestCase
 {
     /**
-     * @expectedException \PhpAmqpLib\Exception\AMQPRuntimeException
+     * @expectedException \ErrorException
```

In the old version stream_socket_client was suppressing errors with `@`. So my test is the way to catch the exception :)